testing/ding-libs/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Vince Mele <vmele@inoc.com>
+# Maintainer: Vince Mele <vmele@inoc.com>
+pkgname=ding-libs
+pkgver=0.6.1
+pkgrel=0
+pkgdesc="'D is not GLib' utility libraries"
+url="https://pagure.io/SSSD/ding-libs/"
+arch="all"
+license="GPL3 LGPL3"
+depends=""
+makedepends="icu-dev gettext-dev doxygen"
```

Comment:
@vincentmele test built this without doxygen with same? results as without it, unused make dependency?